packages/framework/src/resources/workflow/workflow.test.ts

```diff
@@ -99,6 +99,36 @@ describe('workflow function', () => {
     });
   });
 
+  it('should include preferences', async () => {
+    const { definition } = workflow(
+      'setup-workflow',
+      async ({ step }) => {
+        // @ts-expect-error - email subject is missing from the output
+        await step.email('send-email', async () => ({
+          body: 'Test Body',
```

Comment:
No need to keep this `ts-expect-error` directive, it was for a different test.

```suggestion
        await step.email('send-email', async () => ({
          subject: 'Test Subject',
          body: 'Test Body',
```